### PR TITLE
Begin tracking netobserv_ingest_errors as KPI

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves [NETOBSERV-1002](https://issues.redhat.com/browse/NETOBSERV-1002)

Also did the following:
- Removed some duplicate queries and expanded data exposed in Touchstone
- Updated Dittybopper configs
- Increased FLP wait timeout
- Added NetObserv must-gather plugin to Jenkins params

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -632,6 +632,7 @@ pipeline {
                             string(name: 'BUILD_NUMBER', value: params.FLEXY_BUILD_NUMBER),
                             booleanParam(name: 'CERBERUS_CHECK', value: params.CERBERUS_CHECK),
                             booleanParam(name: 'MUST_GATHER', value: true),
+                            string(name: 'IMAGE', value: 'quay.io/netobserv/must-gather'),
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply Jenkinsfile]: @paigerube14 i like it! updated :+1:

**Review by memodi — COMMENTED** [review]
@nathan-weinberg - just one minor comment. I wonder, if we should remove per pod queries altogether for all types of metrics and just keep totals, wdyt?

File: scripts/queries/netobserv_prometheus_queries.yaml [hunk 2]
```diff
@@ -6,10 +6,18 @@
...
+# Total flows errored per minute
+- query: sum(rate(netobserv_ingest_errors[1m])*60)
+  metricName: nFlowsErroredPerMinuteTotals
+
 # Flows processed by ingester per minute aggregated by FLP pods
 - query: sum(rate(netobserv_ingest_flows_processed[1m])*60) by (pod)
   metricName: nFlowsProcessedPerMinute
 
+# Flows errored by ingester per minute aggregated by FLP pods
+- query: sum(rate(netobserv_ingest_errors[1m])*60) by (pod)
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/queries/netobserv_prometheus_queries.yaml:18]: I'd like to keep the per pod metrics - they have come in use, for example for Mohamed's testing of a new eBPF image to see resource utilization across pods. I have noticed that these metrics don't come up if they value out as 0 (such as in the sheet above) so shouldn't bloat ES or the spreadsheets with useless 0 data.

**Pull request merged**